dataactbroker/fileRoutes.py

```diff
@@ -130,6 +130,76 @@ def get_protected_files():
         file_manager = FileHandler(request, is_local=is_local, server_path=server_path)
         return file_manager.get_protected_files()
 
+    @app.route("/v1/check_current_page/", methods=["GET"])
+    @convert_to_submission_id
+    @requires_submission_perms('reader')
+    def check_submission(submission_id):
+        sess = GlobalDB.db().session
+
+        # check if submission_id exists in database
+        check_submission_id = sess.query(Job).filter(Job.submission_id == submission_id)
```

Comment:
If we're checking for the existence of a submission, it should be checked against the `submission` table